### PR TITLE
Push up to date

### PR DESCRIPTION
**Pull request by jgrund**
Add all commits up to master

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+- 6
+cache: yarn
```
Comment (johnsonw) [inline .travis.yml]: Do we not want 7 here as well?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # Flow Highland
```
Comment (johnsonw) [inline README.md]: Should the name be "flow-highland" based on our conversation this morning?

File: .travis.yml (hunk 1 above)
Comment (jgrund) [inline .travis.yml]: We are not running any tests.

File: README.md (hunk 2 above)
Comment (jgrund) [inline README.md]: Yeah, I'll update.

**Pull request merged**